src/utils/api.js

```diff
@@ -31,3 +31,20 @@ export function updateArticleVote(id, votes) {
 export function postNewComment(id, body) {
   return articlesApi.post(`/articles/${id}/comments`, body);
 }
+
+export function getTopics() {
+  return articlesApi
+    .get(`https://catherine-nc-news.onrender.com/api/topics`)
+    .then(({ data }) => {
+      return data.topics;
+    });
+}
+
+export function getArticlesByTopics(topic) {
+  return articlesApi.get("./articles").then(({ data }) => {
+    const filterTopic = data.articles.filter((article) => {
```

Comment:
If you were able to add the query endpoint to your backend during the backend project (ie "/api/review?topic=cats") it would be more efficient than manipulating the data using javascript after all the articles have been returned

Reply:
Thanks for your advice! I will change it;) Thanks for your advice these few days!